src/test/java/th/ac/kmitl/science/comsci/example/CityNameTest.java

```diff
@@ -0,0 +1,16 @@
+package th.ac.kmitl.science.comsci.example.models;
+
+import junit.framework.Assert;
+import org.junit.Test;
+public class CityNameTest {
+    String cname = "Bangkok";
+    CityName ctyname = new CityName(cname);
+    @test
+    public void testgetCityName(){
+        Assert.assertEquals(ctyname.getCityName(),"Bangkok");
```

Comment:
you have one `cityName` variable, so in this assert statement, please compare it with a variable.

`Assert.assertEquals(ctyname.getCityName(), cityName);`